wallet/build.gradle

```diff
@@ -41,63 +42,95 @@ dependencies {
     implementation 'com.google.android:flexbox:1.1.0'
     implementation 'androidx.preference:preference:1.1.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    implementation "androidx.work:work-runtime-ktx:2.4.0"
     implementation "org.dashj:dashj-core:$dashjVersion"
     implementation "org.dashj-android:dashj-bls-android:0.17.5"
     implementation "org.dashj-android:dashj-x11-android:0.17.5"
     implementation "org.dashj-android:dashj-scrypt-android:0.17.5"
+    implementation 'io.grpc:grpc-stub:1.28.0' // CURRENT_GRPC_VERSION
     implementation 'com.google.guava:guava:29.0-android'
-    implementation 'com.google.zxing:core:3.3.3'    // 3.3.3 is the maximum to support Android 4.4 to 6.x
+    implementation 'com.google.zxing:core:3.3.3'
+    // 3.3.3 is the maximum to support Android 4.4 to 6.x
```

Comment:
put the comment back on the previous line

Reply:
Done